admin/js/editor.js

```diff
@@ -32,4 +32,33 @@ wp.domReady(() => {
   ];
 
   registerBlockStyle('core/button', styles);
+
+  ['core/media-text', 'core/group', 'core/column'].forEach(
+    block => registerBlockStyle(block, [
+      {
+        name: 'small-padding',
+        label: 'Small padding',
+      }, {
+        name: 'medium-padding',
+        label: 'Medium padding',
+      }, {
+        name: 'large-padding',
+        label: 'Large padding',
+      },
+    ]),
+  );
+
+  ['core/group'].forEach(block => {
+    registerBlockStyle(block, {
+      name: 'space-evenly',
+      label: __('Space evenly', 'planet4-blocks-backend'),
+    });
+  });
+
+  ['core/paragraph'].forEach(block => {
```

Comment:
This style allows to editor to choose the `Roboto` font family. Also defined through the design system.